src/journal/bsod.c

```diff
@@ -280,7 +298,7 @@ static int run(int argc, char *argv[]) {
                 return 0;
         }
 
-        r = display_emergency_message_fullscreen((const char*) message);
+        r = display_emergency_message_fullscreen((const char*) message, message_id);
```

Comment:
Unnecessary cast. Please drop it.

Reply:
> Unnecessary cast. Please drop it.

Thanks @yuwata for these reviews.
I'll work on them and push.
I hope they make v255 if I can work on them as hard as bluca would.